lib/node_modules/@stdlib/repl/README.md

```diff
@@ -1329,9 +1329,9 @@ Out[2]: [ 'stdlib-ansi-basic', 'myTheme' ]
 
 Starts a tutorial.
 
-```text
-In [1]: var id = tutorial( 'repl' );
-```
+    ```text
+    In [1]: var id = tutorial( 'repl' );
+    ```
```

Comment:
Change unrelated to this PR; should be reverted.

````suggestion
```text
In [1]: var id = tutorial( 'repl' );
```
````